prov/gni/include/gnix.h

```diff
@@ -379,7 +380,8 @@ struct gnix_fab_req_msg {
 struct gnix_fab_req {
 	struct slist_entry        slist;
 	uint64_t tag;
-	uint64_t mask_bits;
+	uint64_t ignore_bits;
+	uint64_t addr_ignore_bits;
```

Comment:
addr ignore is for the wild card EP case. I was under the assumption that a user could specify a set of EPs by ignoring some of the its. This might be a misunderstanding on my part.